database/java/2017/9803.yaml

```diff
@@ -0,0 +1,18 @@
+cve: 2017-9803
+title: "Security vulnerability in kerberos delegation token functionality"
+description: >
+    Solr's Kerberos plugin can be configured to use delegation tokens, which allows an application to reuse the authentication of an end-user or another application. There are two issues with this functionality (when using SecurityAwareZkACLProvider type of ACL provider e.g. SaslZkACLProvider). Firstly, access to the security configuration can be leaked to users other than the solr super user. Secondly, malicious users can exploit this leaked configuration for privilege escalation to further expose/modify private data and/or disrupt operations in the Solr cluster. The vulnerability is fixed from Solr 6.6.1 onwards
```

Comment:
The description would be shorter and clearer if we omit "The vulnerability is fixed from Solr 6.6.1 onwards" from it. We have a dedicated field for such information.

WDYT?

Reply:
@php-coder Agree, this can be shorter
Im going to make another PR later for another CVE I just found, will fix it altogether